Tools/BatchScripts/batch_summit.sh

```diff
@@ -19,5 +19,4 @@ module load cuda
 omp=1
 export OMP_NUM_THREADS=${omp}
 
-num_nodes=$(( $(printf '%s\n' ${LSB_HOSTS} | sort -u | wc -l) - 1 ))
-jsrun -n ${num_nodes} -a 6 -g 6 -c 6 --bind=packed:${omp} <path/to/executable> <input file> > output.txt
+jsrun -a 6 -g 6 -c 6 --bind=packed:${omp} <path/to/executable> <input file> > output.txt
```

Comment:
- current layout: `jsrun -a 6 -g 6 -c 6`: https://jsrunvisualizer.olcf.ornl.gov/?s4f1o01n1c6g6r16d1b21l0= (ill-placed for optimal latency)
- @WeiqunZhang: `jsrun -r 6 -a 1 -g 1 -c 7 --bind=packed:7` https://jsrunvisualizer.olcf.ornl.gov/?s4f1o01n6c7g1r11d1b27l0= (okay as we use WarpX today)

Reply:
I think this could be potentially useful in the future, when using auxiliary OpenMP functionality outside of kernels (e.g. with our already existing BLAS/LAPACK bindings for RZ+PSATD): https://jsrunvisualizer.olcf.ornl.gov/?s4f1o17n6c7g1r11d1b1l0=

See setup in PIConGPU: https://github.com/ComputationalRadiationPhysics/picongpu/blob/0.5.0/etc/picongpu/summit-ornl/gpu_batch.tpl
```
export OMP_NUM_THREADS=7
jsrun -n 6 -a 1 -c 7 -g 1 -r 6 -l GPU-CPU -d packed -b rs --smpiargs="-gpu"
```

Reply:
For some reason, AMReX apps run w/o CUDA aware MPI: https://github.com/AMReX-Codes/amrex/blob/development/Tutorials/GPU/run.summit

Not a good idea imho (see https://github.com/ComputationalRadiationPhysics/picongpu/pull/438), but has probably some implementation reason.